go.mod

```diff
@@ -6,9 +6,9 @@ require (
 	github.com/Pallinder/go-randomdata v1.2.0
 	github.com/PuerkitoBio/goquery v1.8.1
 	github.com/avast/retry-go v3.0.0+incompatible
-	github.com/aws/aws-sdk-go v1.44.326
-	github.com/aws/karpenter-core v0.30.0-rc.0.0.20230816121541-da2c2f5594a3
-	github.com/go-playground/validator/v10 v10.14.1
+	github.com/aws/aws-sdk-go v1.44.327
+	github.com/aws/karpenter-core v0.100.101-test
+	github.com/go-playground/validator/v10 v10.15.1
```

Comment:
From what I could tell, this upgrade breaks us since this validator was never working with `metav1.Duration` to begin with. Before we take this upgrade in, we should remove this dependency.